Makefile.am

```diff
@@ -229,6 +233,15 @@ EXTRA_DIST += \
 	nmsg/base/pkt.c \
 	nmsg/base/xml.c
 
+FLT_LIBTOOL_FLAGS = -module -avoid-version -shared -export-symbols-regex "^nmsg_fltmod_plugin_export$$"
```

Comment:
Could you please provide some documentation for building these modules outside of the nmsg source tree?